hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneAclConfig.java

```diff
@@ -40,6 +40,9 @@
           "OzoneManager."
   )
   public void setUserDefaultRights(String userRights) {
+    if(userRights == null) {
```

Comment:
should we handle this in the getter? if the caller set with null, we should just ignore it instead of changing it to ALL.

Reply:
This results in NPE in MR robot tests. While instantiating RPC client via reflection we get userRights as null which results in NPE.  Also from config point if userRights is null it means user hasn't set any value for it, so it should fallback to default.

Reply:
Make sense to me. But should put the real "defaultValue" instead of hard-coded  "ALL".